docker/centos-7/Dockerfile

```diff
@@ -1,22 +1,22 @@
 FROM centos:7
 
-RUN rpm --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
-RUN yum -y install wget
-RUN yum -y install curl
-RUN yum -y install make m4 gcc patch unzip
-RUN yum -y install git rsync mercurial
-RUN yum -y install gcc-c++
-RUN yum -y install bzip2-devel libffi-devel snappy-devel libev-devel
-RUN yum -y install python-devel
-
-# TODO: What about darcs ?
-
 RUN useradd jenkins -u 1500 -g root
 
+RUN rpm --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
 RUN rpm -iUvh http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-10.noarch.rpm
-RUN yum -y update && \
-    yum -y install python-pip openssl-devel gmp-devel which zlib-devel \
-                   ncurses-devel bzip2 cmake3 sudo
+
+RUN yum -y update \
```

Comment:
FYI messed around with this part because I wanted to add the 'file' dependency (some configure scripts complain about it - but carry on nonetheless), and because the yum update & install should be done in the same docker step